# Robert S. Harbert, Ph.D.
Email: rsh249@cornell.edu
Phone: 540-354-8104
LinkedIn: https://www.linkedin.com/in/robharbert 
Publications: Rob Harbert (0000-0002-1714-5033) - ORCID
GitHub: https://github.com/rsh249 http://github.com/developingbioinformatics

# Overview
Solving technical challenges to make better science happen. I am an experienced bioinformatician with expertise in R and python programming, statistics and modeling, and DNA sequencing analysis focused on delivering top tier bioinformatics support and insights for biotechnology platforms. I am experienced in both software service and support and development in bioinformatics. 

# Experience
## Bioinformatics Scientist – Bruker Cellular Analysis – June 2022 – present
Critical link between software development and customer support teams. Solve customer issues in bioinformatics pipeline application and data. Conducted root cause investigation and lead escalation team meetings to solve critical customer use issues in an NGS antibody discovery pipeline in 6 months. Analysis of single cell BCR-seq and integration with fluorescent assay and other instrument data.
Skills: customer support, software support, Salesforce case and work order management, NGS and Sanger data analysis and QC, RNAseq, data visualization, antibody discovery, molecular biology.
## Genomic Data Science, Sr. Specialist – PharmaLex – 2021-2022
Building bioinformatics solutions in a fast paced consulting team. Programming solutions in pharmaceutical research for statistical genomics (GWAS, meta-analyses, PRS, MR), single cell and bulk RNA-seq including cell type assignments, DE, and trajectory analyis, and clinical data analysis. 
Skills: R programming, Spark, data visualization, documentation and validation.
## Assistant Professor of Biology and Bioinformatics – Stonehill College – 2018-2021
Teaching undergraduate and graduate classes in biology, botany, bioinformatics, statistics, and data analytics. Research in metagenomics, paleogenomics, and nanopore sequencing of environmental DNA. 
Skills: NGS, nanopore sequencing, DNA extraction and isolation from degraded and non-cellular sources, R and Python programming.
## Adjunct Professor of Data Analytics — Stonehill College — 2020-2021	
Statistics and big-data analytics course teaching in the Data Analytics masters degree program. 
Skills: SAS programming, statistics, big-data analytics, Spark, pyspark.
## Gerstner Scholar in Computational Biology and Bioinformatics – American Museum of Natural History – 2016-2018
Postdoctoral scholarly research and consulting on computational projects. Owned research projects focused on ancient DNA, computational methods for metagenomic identification of plants, and computational paleoclimatology. 
Skills: DNA extraction, metagenomic methods development, modeling and visualization, Linux, HPC, bash scripting.

# Volunteer
Mentor — NECyberteam — 2019-2021 	Advisor for computational projects run by participating faculty. Worked directly with student cyberteam workers 			to guide the scope of work and implementation of technical infrastructure.
Visiting Scientist — American Museum of Natural History — 2018-2023 	Scientific collaborator and advisory expert in computing and ancient DNA technology.

# Education
Ph.D., Plant Biology – 2016 – Cornell University, Ithaca, NY
Scholarly research in plant evolutionary ecology and paleoclimatology. Developing CRACLE: https://github.com/rsh249/cRacle 

B.S. Biology – 2011 – Roanoke College, Salem, VA

# Highlights
**Programming:** R, python, Spark, bash/shell scripting.
**Other Tech:** High-performance and cloud computing, R package development, Nextflow, parallelization techniques, Docker, Git, SQL, Unix/Linux operating systems management.
**Bioinformatics: **Assembly and annotation of antibody sequences, whole genome assembly and variant calling, metagenomic analysis of environmental samples, genomic pipelines, statistical genomics including GWAS, PRS, and MR analyses.
**Lab Experience:** Molecular biology, DNA extraction, PCR, sequencing technologies including Nanopore devices.

# Academic Publications

Harbert, R., Cunningham, S.W. and Tessler, M., 2020. Spatial modeling could not differentiate early SARS-CoV-2 cases from the distribution of humans on the basis of climate in the United States. PeerJ, 8, p.e10140.

Moore, G., Tessler, M., Cunningham, S.W., Betancourt, J. and R. Harbert. 2020. Paleo‐metagenomics of North American fossil packrat middens: Past biodiversity revealed by ancient DNA. Ecology and Evolution, 10(5), pp.2530-2544. https://doi.org/10.1002/ece3.6082 

Harbert, R.S., and A. Baryiames. 2019. cRacle: R Tools for Estimating Climate from Vegetation. Applications in Plant Sciences, e11322. https://doi.org/10.1002/aps3.11322

Harbert, R.S., and K.C. Nixon. 2018. Dynamic Quaternary vegetation as a proxy for 50,000 years of climate change in Western North America. Open Quaternary, https://doi.org/10.5334/oq.46  

Harbert, R.S. 2018. Algorithms and database strategy for the reconstruction of plant communities from ancient and environmental DNA. Applications in Plant Sciences, e1034 

Martinez, C., T.Y.S. Choo, D. Allevato, K. Nixon, W. Crepet, R. Harbert, C. Daghlian. 2016. Rariglanda jerseyensis a new ericalean fossil flower from the Late Cretaceous of New Jersey. Botany 94: 747–758 dx.doi.org/10.1139/cjb-2016-0062  

Harbert, R.S., and K.C. Nixon. 2015. Climate reconstruction analysis using coexistence likelihood estimation (CRACLE): A method for the estimation of climate using vegetation. American Journal of Botany, doi:10.3732/ajb.1400500  

Harbert, R.S., A.H.D. Brown, and J. Doyle. 2014. Climate Niche Modeling in the Perennial Glycine (Leguminosae) Allopolyploid Complex. American Journal of Botany 101(4):710-721.  

Pysh, L., N. Alexander, L. Swatzyna, and R. Harbert. 2012. Four alleles of AtCESA3 form an allelic series with respect to root phenotype in Arabidopsis thaliana. Physiologia Plantarum 144:369-381.  
